pkg/asset/installconfig/openstack/validation/cloudinfo.go

```diff
@@ -181,13 +180,9 @@ func (ci *CloudInfo) collectInfo(ic *types.InstallConfig, opts *clientconfig.Cli
 }
 
 func (ci *CloudInfo) getSubnet(subnetID string) (*subnets.Subnet, error) {
-	if subnetID == "" {
-		return nil, nil
-	}
```

Comment:
this was providing a nice shortcut to exit earlier and safe some CPU cycles here and there...

Reply:
+1 also not sure why this check needs to be removed. Otherwise lgtm

Reply:
yeah, let's keep it